addons/block_code/ui/constants.gd

```diff
@@ -24,130 +24,156 @@ const BUILTIN_CATEGORIES_PROPS: Dictionary = {
 	{
 		"color": Color("ec3b59"),
 		"order": 10,
+		"icon": "lifecycle",
```

Comment:
Below in the PR you do this for the button tooltip text:
```
_button.tooltip_text = tr(category.name.get_slice(" |", 0))
```

So I think it's better to do the same thing instead of introducing a new "icon" key in this constant. If you want to keep the SVG files lowercase, you can use [to_lower()](https://docs.godotengine.org/en/stable/classes/class_string.html#class-string-method-to-lower)